extensions/NOname-awa/More comparisons

```diff
@@ -0,0 +1,750 @@
+                            },
+                            {
+                                text: formatMessage({
+                                    id: 'EXTENSION.TYPE_EVEN_NUMBER',
+                                    default: 'even number'
+                                }),
+                                value: 'evenNumber'
+                            },
+                            {
+                                text: formatMessage({
+                                    id: 'EXTENSION.TYPE_ODD_NUMBER',
+                                    default: 'odd number'
+                                }),
+                                value: 'oddNumber'
+                            },
+                            {
+                                text: formatMessage({
+                                    id: 'EXTENSION.TYPE_INTEGER_NUMBER',
+                                    default: 'integer number'
+                                }),
+                                value: 'integerNumber'
+                            },
+                            {
+                                text: formatMessage({
+                                    id: 'EXTENSION.TYPE_DECIMAL_NUMBER',
+                                    default: 'decimal number'
+                                }),
+                                value: 'decimalNumber'
+                            },
+                            {
+                                text: formatMessage({
+                                    id: 'EXTENSION.TYPE_PRIME_NUMBER',
+                                    default: 'prime number'
+                                }),
+                                value: 'primeNumber'
+                            },
+                            {
+                                text: formatMessage({
+                                    id: 'EXTENSION.TYPE_COMPOSITE_NUMBER',
+                                    default: 'composite number'
+                                }),
+                                value: 'compositeNumber'
+                            },
+                            {
+                                text: formatMessage({
+                                    id: 'EXTENSION.TYPE_TEXT',
+                                    default: 'text'
+                                }),
+                                value: 'text'
+                            },
+                            {
+                                text: formatMessage({
+                                    id: 'EXTENSION.TYPE_BOOLEAN',
+                                    default: 'boolean'
+                                }),
+                                value: 'boolean'
+                            }
+                        ]
+                    }
+                }
+            };
+        }
+        booleanMenu({ BOOLEAN }) {
+            if (BOOLEAN === 'random') return Math.random() < 0.5 ? true : false
```

Comment:
You don't have to do this, but internally Scratch's "random" value is `_random_`, consider changing to that for consistency